src/main/java/edu/cornell/kfs/sys/xmladapters/StringToJavaDateAdapter.java

```diff
@@ -16,7 +17,7 @@
  */
 public class StringToJavaDateAdapter extends XmlAdapter<String, Date> {
 
-    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormat.forPattern("MM/dd/yyyy");
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormat.forPattern("MM/dd/yyyy").withLocale(Locale.US);

```

Comment:
can the format string be moved to a new constant in CUKFSConstants

Reply:
Constant created.